lib/oneview-sdk/image-streamer/resource/api300/build_plan.rb

```diff
@@ -17,6 +17,22 @@ module API300
       # Build Plan resource implementation for Image Streamer
       class BuildPlan < Resource
         BASE_URI = '/rest/build-plans'.freeze
+
+        # Create a resource object, associate it with a client, and set its properties.
+        # @param [OneviewSDK::ImageStreamer::Client] client The client object for the Image Streamer appliance
+        # @param [Hash] params The options for this resource (key-value pairs)
+        # @param [Integer] api_ver The api version to use when interracting with this resource.
+        def initialize(client, params = {}, api_ver = nil)
+          super
+          # Default values:
+          @data['type'] ||= 'OeBuildPlan'
+        end
+
+        # Build step of the build plan.
+        # @param build_step [Array]  The Build step array of the build plan
+        def set_build_steps(build_step = [])
```

Comment:
I thought it was best this way. But his suggestion is also good. I'll change it.